GeoHealthCheck/config_main.py

```diff
@@ -47,6 +47,7 @@
 GHC_NOTIFICATIONS_EMAIL = ['you2@example.com']
 GHC_SITE_TITLE = 'GeoHealthCheck Demonstration'
 GHC_SITE_URL = 'http://host'
+GHC_METADATA_CACHE_SECS = 7200
```

Comment:
Not sure. I agree that things can be cleaned up a bit. The main issue is that we run GHC both as a webapp and for cronjobs, so some scripts need access to the DB/ORM.